tools/conversion/keras_converter.py

```diff
@@ -99,7 +99,7 @@ def create_keras_model(self):
             outputs=[self.container.all_layers[i] for i in self.container.out_index],
         )
         logging.info("done assembling model")
-        logging.info(model.summary())
+        model.summary(print_fn=logging.info)
 
         # logging.info() all inputs, formatted for use with coremltools Keras convertor
         logging.info("input_names=[")
```

Comment:
Can we indent the ones inside the loop? With each of these on a new line it's a bit hard to see where it starts/ends:

```
INFO:root:input_names=[
INFO:root:'new_frame',
INFO:root:'frame_back_1',
INFO:root:'frame_back_2',
INFO:root:'frame_back_3',
INFO:root:'cnn.3_history_0',
INFO:root:'cnn.3_history_1',
INFO:root:'cnn.7_history_0',
INFO:root:'cnn.11_history_0',
INFO:root:'cnn.11_history_1',
INFO:root:'cnn.14_history_0',
INFO:root:'cnn.17_history_0',
INFO:root:'cnn.17_history_1',
INFO:root:'cnn.20_history_0',
INFO:root:'cnn.20_history_1',
INFO:root:'cnn.23_history_0',
INFO:root:'cnn.23_history_1',
INFO:root:'cnn.25_history_0',
INFO:root:'cnn.25_history_1',
INFO:root:],
```

Reply:
Thanks. I've added the indentation and actually reversed these lines to use `print()` and not `logging.info()`. I think these lines can be considered actual program output that should be displayed _always_, no matter if `--verbose` is set or not.